src/megaclient.cpp

```diff
@@ -6152,6 +6157,43 @@ void MegaClient::sc_la()
     }
 }
 
+void MegaClient::sc_ub()
+{
+    bool statusRecv = false;
+    int status = -2;
+    for (;;)
+    {
+        switch (jsonsc.getnameid())
+        {
+            case 's':
+                statusRecv = true;
+                status = (int)jsonsc.getint();
+                break;
+
+            case EOO:
+                if (!statusRecv)
+                {
+                    std::string err = "Missing status in `ub` action packet";
+                    LOG_warn << err;
```

Comment:
This is clearly an error, not a simple warning. It would cause issues and un-expected behavior, so better to state it as Error in the log message.
```suggestion
                    LOG_err << err;
```